### PR TITLE
Add getIcon and setIcon to INotification

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description
This adds an icon to the notification.

## Related Issue
https://github.com/owncloud/notifications/issues/102

## Motivation and Context
Make notification great again!

## How Has This Been Tested?
I'm still adding changes to the notifications app ... will update soon

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: lib/private/Notification/Notification.php [hunk 1]
```diff
@@ -450,4 +450,25 @@ protected function isValidCommon() {
 			$this->getObjectId() !== ''
 		;
 	}
+
+	/**
+	 * Icon is an absolute URL to an image to be displayed in the notificationss
```
Comment (phil-davis) [inline lib/private/Notification/Notification.php]: notificationss

File: lib/public/Notification/INotification.php [hunk 2]
```diff
@@ -220,4 +220,18 @@ public function isValid();
 	 * @since 9.0.0
 	 */
 	public function isValidParsed();
+
+	/**
+	 * Icon is an absolute URL to an image to be displayed in the notificationss
```
Comment (phil-davis) [inline lib/public/Notification/INotification.php]: notificationss

Comment (PVince81) [inline lib/public/Notification/INotification.php]: huh, what'ssss wrong ? 🐍

**Review by PVince81 — APPROVED** [review]
:+1:

Assuming this won't break any apps since INotification is always instantiated internally.

**Pull request merged**